stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.1.0-a
+version: 1.2.0-a
```

Comment:
Thanks @krancour, I'll update this to be `1.1.0-b` (although I'm a little concerned this may affect version comparisons since `-` in the SemVer spec is for pre-releases - something to think about separately.). 

If #257 gets merged in before this, I'll update the version here.